homeassistant/components/google_sheets/const.py

```diff
@@ -1,10 +1,22 @@
 """Constants for Google Sheets integration."""
 from __future__ import annotations
 
+from enum import Enum
 from typing import Final
 
 DOMAIN = "google_sheets"
 
+CONF_SHEETS_ACCESS = "sheets_access"
 DATA_CONFIG_ENTRY: Final = "config_entry"
 DEFAULT_NAME = "Google Sheets"
-DEFAULT_ACCESS = "https://www.googleapis.com/auth/drive.file"
+
+
+class FeatureAccess(Enum):
```

Comment:
This would be best as a `StrEnum` (`from homeassistant.backports.enum import StrEnum`), so then you don't need to use `.value` to get the string constant.